packages/amplify_core/lib/src/config/analytics/pinpoint_config.dart

```diff
@@ -46,16 +48,26 @@ class PinpointPluginConfig
   final PinpointAnalytics pinpointAnalytics;
   final PinpointTargeting pinpointTargeting;
 
+  /// The duration in seconds (rounded up) between flushing analytics events to Pinpoint.
+  ///
+  /// Max events per flush is 100.  Decrease duration to ensure you flush more events than you store over time.
```

Comment:
> (rounded up)

I believe it will be rounded down, not up. The rounding behavior is only applicable if this is serialized to JSON.

> Max events per flush is 100. Decrease duration to ensure you flush more events than you store over time.

I'm not sure that you need to include this, but if you do, this probably needs a bit more detail. I could see someone not understanding that events will queue on disk. The duration only needs to be decreased if there will be a *sustained* high rate of events. Small bursts at a high rate would likely be fine for most use cases as they would just queue on disk and be sent shortly after.

```suggestion
  /// The duration between flushing analytics events to Pinpoint.
  ///
  /// A max of 100 events are included in each request. If more than 100 events 
  /// are recorded per interval, they will be temporarily queued and sent in a 
  /// later request. Consider reducing the duration for use cases where events 
  /// are recorded at a high rate for sustained periods of time.
  ///
  /// *Note:* The duration will be rounded down to the nearest whole second when 
  /// this value is serialized to json.
```